DSCResources/MSFT_xHostsFile/MSFT_xHostsFile.psm1

```diff
@@ -0,0 +1,267 @@
+$Script:hostsFilePath = "${env:windir}\system32\drivers\etc\hosts"
+
+Import-LocalizedData -BindingVariable LocalizedData -filename MSFT_xHostsFile.psd1 -BaseDirectory $PSScriptRoot -Verbose
+
+function Get-TargetResource
+{
+    [OutputType([Hashtable])]
+    param (
+        [parameter(Mandatory = $true)]
+        [string]
+        $HostName,
+        [parameter(Mandatory = $true)]
+        [string]
+        $IPAddress
+    )
+
+    $configuration = @{
+        HostName = $HostName
+        IPAddress = $IPAddress
+    }
+
+    Write-Verbose $localizedData.checkingHostsFileEntry
+    try {
```

Comment:
PowerShell Studio 2015 can automatically correct this for you automatically, but it isn't something I'd move over to just for that feature (I love VS Code). A PSScriptAnalyzer rule would be pretty useful if one could be created - it could be added to the Meta.Tests.ps1 and run as part of the base tests.

I am also sure with a little bit of effort a simple "fixer" could also be created that could be added to Meta.Fixers.ps1.

I'll add this as an Issue so it can be discussed.